### PR TITLE
Remove references to PlatformAbstractions

### PR DESCRIPTION
**Pull request by eerhardt**
PlatformAbstractions is being obsoleted and will be removed from dotnet/runtime.

Contributes to https://github.com/dotnet/runtime/issues/3470

### CODE REVIEW

File: dir.props [hunk 1]
```diff
@@ -213,4 +224,4 @@
     <ExtraPackageVersionPropsPackageInfo Include="MicrosoftSourceLinkVersion" Version="%24(MicrosoftSourceLinkCommonPackageVersion)" />
   </ItemGroup>
 
-</Project>
+</Project>
```
Comment (dseefeld) [inline dir.props:227]: Nit: no newline at the end of the file.

File: tools-local/init-build.proj [hunk 2]
```diff
@@ -171,4 +131,4 @@
     <Message Importance="High" Text="[$([System.DateTime]::Now.ToString('HH:mm:ss.ff'))] Done writing out SourceLink metadata." />
   </Target>
 
-</Project>
+</Project>
```
Comment (dseefeld) [inline tools-local/init-build.proj:134]: Nit: no newline at the end of the file.

**Comment by dseefeld** [comment]
It looks like the CI failures are all prebuilt baseline updates.  These should be able to just be updated and CI will pass.

**Comment by eerhardt** [comment]
> These should be able to just be updated and CI will pass.

Is there an automated way of updating them? If not, where are they so I can update them?

**Pull request merged**